### DISCUSSION TITLE
Buggy Vertical Scroll Bar present on the right when and only when using the Matix Rain wallpaper

### DISCUSSION THREAD

**Discussion by xian-su** [rocksdanister/lively#1992] [General]
**Describe the bug**
Buggy Vertical Scroll Bar present on the right when and only when using the Matix Rain theme

**To Reproduce**
Whenever apply the Matrix Rain theme, the scroll bar appears

**Expected behavior**
Scroll bar not to be there

**Screenshots/Video**
Please see the attached image
![image](https://github.com/rocksdanister/lively/assets/150081303/37607535-98f7-426b-b92e-55cbd9d84fda)

**Desktop (please complete the following information):**
 - OS: Win11 22H2
- Only with the Matrix Rain wallpaper

**Additional context**
NA

**Log file (Important)**
Share the diagnostic files by:
Open Lively settings (Gear icon) top-right -> System -> Log File -> Save
or 
Navigate to Appdata folder `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`

**Comment by rocksdanister** [comment]
Add this to the css style of wallpaper:
https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp

**Reply by xian-su** [reply]
Thank you so much! It worked~
